src/chat.cpp

```diff
@@ -1131,6 +1225,152 @@ CCMD( say_team )
 	}
 }
 
+//*****************************************************************************
+//
+// [AK] Allows players (or the server) to send private messages to other players.
+//
+bool chat_CanSendPrivateMessage( ULONG ulPlayer )
+{
+	if ( ulPlayer == MAXPLAYERS )
```

Comment:
We don't need to make sure g_ulChatPlayer is a valid target if the sender is the server?

Reply:
You're right, that needs to checked as well.